docker-entrypoint.sh

```diff
@@ -10,12 +10,12 @@ set -e
 #   database = postgres://...
 
 for var in "${!TRAC_INI_@}"; do
-    sed -i "s;^${var:9} = .*;${var:9} = ${!var};" trac-env/conf/trac.ini
+    sed -i "s;^${var:9} = .*;${var:9} = ${!var};" /code/trac-env/conf/trac.ini
 done
 
 if [ "x$TRAC_COLLECT_STATIC" = 'xon' ]; then
```

Comment:
For the future -- perhaps we could use a similar trick to allow upgrade to be run more easily, when it is needed? (Not now.)